frequently-asked-questions.md

```diff
@@ -0,0 +1,90 @@
+# Frequently Asked Questions (FAQ)
+
+This is a list of the most frequently asked questions by members of this community.
+
+## Engaging with Community
+
+**Q: I am new to open source. Where should I start contributing?**
+A: If it's your first time contributing to a project, please build and explore [AnitaB.org Open Source](https://github.com/anitab-org). Some repositories have some "first timers only" issues for newcomers. For example, Mentorship Backend contains [a list of “first timers only” issues](https://github.com/anitab-org/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22First+Timers+Only%22) regarding both Documentation and Quality Assurance.
+
+**Q: What is and who can join Community Open Sessions?**
+A: These are sessions where anyone interested in discussing community concerns or projects can join regardless of title, membership level, repository access, or current program involvement.
+
+**Q: What is the minimal technical knowledge needed to contribute to AnitaB.org projects?**
+A: **Zero technical knowledge needed!** Because you can always help with documentation, testing the app, creating wireframes to update it to the new branding guidelines, share our work through creating online content such as blog posts or posting on social media.
+
+**Q: As a newcomer, how can I contribute to this community?**
+- Help writing notes in Community Open Sessions
+- Write new or improve current documentation
+- Perform Code Reviews on Pull Requests
+- Test a project feature and report bugs found through issues
+- Share feedback on features, UI, UX, …
+- Suggest improvements or new features
+- Design UI/UX improvements
+- Help community members with troubleshooting problems on Zulip and GitHub
+- Design solution architecture
+- Solve available issues on GitHub
+
+You can also check out more ways to contribute on [ways-to-contribute.md](/ways-to-contribute.md).
+
+**Q: I have found a bug. What should I do?**
+A: Raise an issue and wait for the approval. Once it’s approved please make it clear for others you are working on the issue.
+
```

Comment:
Nothing big but on Grammarly, this sentence was shown more accurate - 'Once it's approved, please make it apparent for others you are working on that issue.'

Reply:
This makes it sound like the contributor has to make it clear that he/she working on the issue but generally following the community guidelines one has to be assigned to start working on a issue approval. 
So we can probably frame something which denotes that you can comment if you are interested to work on the issue and will be assigned soon in a bit proper English 
What say ? @isabelcosta @aditmehta

Reply:
Yes you are right. I agree.

Reply:
nice point @vj-codes , I just changed it, let me know what you think :)